src/cpp/session/modules/SessionConsole.R

```diff
@@ -14,3 +14,71 @@
 #
 #
 
+.rs.addJsonRpcHandler("console_follow_hyperlink", function(url, text, params)
+{
+  if (!grepl("://", url)) {
+    stop("missing scheme")
+  }
+
+  if (identical(params, "")) {
+    params <- list()
+  } else {
+    params <- strsplit(params, ":")
+
+    names  <- sapply(params, function(p) sub("=.+$", "", p))
+    params <- lapply(params, function(p) sub("^[^=]+=", "", p))
+    names(params) <- names
+  }
+  
+  scheme <- sub("://.*$", "", url)
+  handler <- switch(scheme, 
+    # rstatshelp:://stats/rnorm
+    # rstatshelp:://stats::rnorm
+    rstatshelp = function(url) {
```

Comment:
From here: https://github.com/rstudio/rstudio/pull/10299#issuecomment-1008694809 

I'm not convinced either, and we do need to think it through because these uri would be generated from user code, probably one of https://github.com/r-lib/cli/issues/390

Another option could be to use the urls used by R, i.e. `http://localhost:8787/help/library/stats/html/rnorm.html`, and redirect that to the help pane. One bonus of this would be that the link would be usable by other cli.